pkg/asset/manifests/tectonic.go

```diff
@@ -60,23 +62,25 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 	}
 
 	assetData := map[string][]byte{
-		"99_binding-discovery.yaml":                             []byte(content.BindingDiscovery),
-		"99_kube-addon-00-appversion.yaml":                      []byte(content.AppVersionKubeAddon),
-		"99_kube-addon-01-operator.yaml":                        applyTemplateData(content.KubeAddonOperator, templateData),
-		"99_openshift-cluster-api_cluster.yaml":                 clusterk8sio.Raw,
-		"99_openshift-cluster-api_worker-machineset.yaml":       worker.MachineSetRaw,
-		"99_openshift-cluster-api_worker-user-data-secret.yaml": worker.UserDataSecretRaw,
-		"99_role-admin.yaml":                                    []byte(content.RoleAdmin),
-		"99_role-user.yaml":                                     []byte(content.RoleUser),
-		"99_tectonic-ingress-00-appversion.yaml":                []byte(content.AppVersionTectonicIngress),
-		"99_tectonic-ingress-01-cluster-config.yaml":            applyTemplateData(content.ClusterConfigTectonicIngress, templateData),
-		"99_tectonic-ingress-02-tls.yaml":                       applyTemplateData(content.TLSTectonicIngress, templateData),
-		"99_tectonic-ingress-03-pull.json":                      applyTemplateData(content.PullTectonicIngress, templateData),
-		"99_tectonic-ingress-04-svc-account.yaml":               []byte(content.SvcAccountTectonicIngress),
-		"99_tectonic-ingress-05-operator.yaml":                  applyTemplateData(content.TectonicIngressControllerOperator, templateData),
-		"99_tectonic-system-00-binding-admin.yaml":              []byte(content.BindingAdmin),
-		"99_tectonic-system-01-ca-cert.yaml":                    applyTemplateData(content.CaCertTectonicSystem, templateData),
-		"99_tectonic-system-02-pull.json":                       applyTemplateData(content.PullTectonicSystem, templateData),
+		"99_binding-discovery.yaml":                              []byte(content.BindingDiscovery),
+		"99_kube-addon-00-appversion.yaml":                       []byte(content.AppVersionKubeAddon),
+		"99_kube-addon-01-operator.yaml":                         applyTemplateData(content.KubeAddonOperator, templateData),
+		"99_openshift-cluster-api_cluster.yaml":                  clusterk8sio.Raw,
+		"99_openshift-cluster-api_worker-machineset.yaml":        worker.MachineSetRaw,
+		"99_openshift-cluster-api_worker-user-data-secret.yaml":  worker.UserDataSecretRaw,
+		"99_openshift-cluster-api_master-machines.yaml":          master.MachinesRaw,
+		"99_openshift-cluster-api_master-user-data-secrets.yaml": master.UserDataSecretsRaw,
```

Comment:
Can we keep this alphabetized by putting these two master entries above the worker entries?

Reply:
not sure why github didn't mark this resolved, but it is done